clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -4799,9 +4812,7 @@ void SYCLIntegrationHeader::setCallsThisGroup(bool B) {
   K->FreeFunctionCalls.CallsThisGroup = B;
 }
 
-SYCLIntegrationHeader::SYCLIntegrationHeader(bool _UnnamedLambdaSupport,
-                                             Sema &_S)
-    : UnnamedLambdaSupport(_UnnamedLambdaSupport), S(_S) {}
+SYCLIntegrationHeader::SYCLIntegrationHeader(Sema &_S) : S(_S) {}
```

Comment:
```suggestion
SYCLIntegrationHeader::SYCLIntegrationHeader(Sema &S) : S(S) {}
```
Let's not use a reserved identifier.